test/TestExpectations.json

```diff
@@ -2257,8 +2257,7 @@
     "testIdPattern": "[network.spec] network Network Events Page.Events.RequestServedFromCache for script",
     "platforms": ["darwin", "linux", "win32"],
     "parameters": ["firefox", "webDriverBiDi"],
-    "expectations": ["FAIL", "PASS"],
-    "comment": "On Firefox, the responseCompleted event for the cached script might be emitted too late (Bug 1907589)"
+    "expectations": ["PASS"]
```

Comment:
@juliandescottes those entries should be removed given that the default expectation is `PASS`.

Reply:
ah yes indeed.